### DISCUSSION TITLE
Original screen not restored on Windows

### DISCUSSION THREAD

**Discussion by peloy** [noborus/ov#166] [General]
Hi Noboru,

There is different behavior on Windows compared to Linux. I can't tell if the problem is in tcell or in oviewer code.

The problem can be seen with ov:

Linux: ov -F small-file.txt

Windows: ov.exe -F small-file.txt

On Linux, the file is displayed and the original screen is restored after ov quits, with the contents of small-file.txt at the bottom of the screen, after the original screen content.

On Windows, the screen is cleared and small-file.txt is displayed at the top of the screen. Then ov quits and the contents of small-file.txt remain at the top of the screen and the original screen content is gone.

Cheers,

Eloy Paris.-

**Comment by noborus** [comment]
Hi,

I think this is a specification of tcell.
Whether tcell creates a new screen or uses the original screen depends on the terminal.
In windows terminal, if you go back to the scroll buffer, you will see the screen that was displayed (not beautiful) until now.
You may also find https://github.com/gdamore/tcell/issues/428 helpful.
